docs/_interface/FAQ.md

```diff
@@ -16,9 +16,9 @@ sort_key: C
 
 ---
 
-**Q.** How do Ethereum syncing work?
+**Q.** How does Ethereum syncing work?
 
-**A.** The current default mode of sync for Geth is called fast sync. Instead of starting from the genesis block and reprocessing all the transactions that ever occurred (which could take weeks), fast sync downloads the blocks, and only verifies the associated proof-of-works. Downloading all the blocks is a straightforward and fast procedure and will relatively quickly reassemble the entire chain.
+**A.** The current default syncing mode used by Geth is called "fast sync". Instead of starting from the genesis block and processing all the transactions that ever occurred (which could take weeks), fast sync downloads the blocks, and only verifies the associated proof-of-works. Downloading all the blocks is a straightforward and fast procedure and will relatively quickly reassemble the entire chain.
```

Comment:
We can also update "fast sync" to say "snap sync"

Reply:
I addressed snap sync in [a new commit](https://github.com/ethereum/go-ethereum/pull/24812/commits/c90e90fe438e4742943d35d030478174b930b114).